tests/CLI/CLI.h

```diff
@@ -48,4 +48,11 @@ std::ostream& operator<<(std::ostream& os, const Date& dt)
     return os;
 }
 
-DLL_API void testFreeFunction();
\ No newline at end of file
+DLL_API void testFreeFunction();
+
+struct CompleteIncompleteStruct;
```

Comment:
That's the point. It can't be used and if it would be compiled the result would be wrong and not useable.